docs/src/pages/components/data-grid/filtering/filtering.md

```diff
@@ -5,15 +5,86 @@ components: DataGrid, XGrid
 
 # Data Grid - Filtering
 
-<p class="description">Filtering helps view particular or related records in the Data Grid.</p>
+<p class="description">Filtering helps view particular or related records in the data grid.</p>
 
-## 🚧 Column filter
+## Column filters
 
-> ⚠️ This feature isn't implemented yet. It's coming.
->
-> 👍 Upvote [issue #201](https://github.com/mui-org/material-ui-x/issues/201) if you want to see it land faster.
+### Basic filter
+
+Column filters can be set using the column menu and clicking the filter menu item.
+Alternatively, if the grid has the toolbar displayed, just need to click on the filter button.
+
+A filtered column can be can pre-configured using the `filterModel` prop:
+
+{{"demo": "pages/components/data-grid/filtering/BasicFilteringGrid.js", "bg": "inline"}}
+
+### Toolbar
+
+In addition to the column menu that allows users to apply a filter, you can also show a toolbar:
+
+{{"demo": "pages/components/data-grid/filtering/BasicToolbarFilteringGrid.js", "bg": "inline"}}
+
+### Disable filter
+
+#### Globally
+
+Filters are enabled by default, but you can easily disable this feature by setting the `disableColumnFilter` prop.
+
+```jsx
+<DataGrid disableColumnFilter />
+```
+
+#### Per column
+
+You can disable the filter on a column by setting the `filterable` property of the `ColDef` to `false`;
+
+```js
+const columns = [{ field: 'image', filterable: false }];
+```
+
+{{"demo": "pages/components/data-grid/filtering/DisableFilteringGrid.js", "bg": "inline"}}
+
+### Custom filter operator
+
+The data grid supports different operators for the native column types.
+However, you can extend the operator and add your own, customize the input component or set your own operator for a new column type.
+
+In this demo, you will see how to reuse the numeric filter and customize the input filter value component.
+
+{{"demo": "pages/components/data-grid/filtering/CustomRatingFilterOperator.js", "bg": "inline"}}
+
+<!-- Demo filter with new column type
+
+Demo filter with new operator -->
+
+<!-- ### Server-side filter -->
+
+<!-- ### Controlled filtering -->
+
+### Multi-filtering ⚡️
```

Comment:
```suggestion
### Multi-column filtering ⚡️
```

Reply:
It can be two filters on the same column, does the proposed title covers it?

Reply:
Hmm... not directly, but "multi-filtering" doesn't read well. Perhaps we mention multiple filters on the same column on the description?